### PR TITLE
[Doc] Add Chinese for modelzoo

### PR DESCRIPTION
**Pull request by xvjiarui**

### CODE REVIEW

File: docs_zh-CN/model_zoo.md [hunk 1]
```diff
@@ -2,156 +2,144 @@
...
-  Otherwise, for input size of 8x (e.g. 512, 1024), `align_corner=False` is adopted.
+* 我们默认使用 4 卡分布式训练
+* 所有 PyTorch 风格的 ImageNet 预训练网络由我们自己训练，和 [论文](https://arxiv.org/pdf/1812.01187.pdf) 保持一致。
+  我们的 ResNet 网络是基于 ResNetV1c 的变种，在这里输入层的 7x7 卷积被 3个 3x3 取代。
+* 为了在不同的硬件上保持一致，我们以 `torch.cuda.max_memory_allocated()` 的最大值作为 GPU 占用率，同时设置 `torch.backends.cudnn.benchmark=False`。
+  注意，这通常比 `nvidia-smi` 显示的要少。
+* 我们以网络 forward 和后处理的时间加和作为推理时间，除去数据加载时间。我们使用脚本 `tools/benchmark.py` 来获取推理时间，它在 `torch.backends.cudnn.benchmark=False` 的设定下，计算 200 张图片的平均推理时间。
+* 在框架中，有两种推理模式。
+  * `slide` 模式（滑动模式）：测试的配置文件字段 `test_cfg` 会是 `dict(mode='slide', crop_size=(769, 769), stride=(513, 513))`.
+    在这个模式下，多个小图分别输入网络中。小图的大小和小图之间的距离由 `crop_size` 和 `stride` 决定，重合区域会进行平均。
```
Comment (Junjun2016) [inline docs_zh-CN/model_zoo.md:13]: 多个小图分别输入网络中 --> 从原图中裁剪多个小图分别输入网络中进行推理

File: docs_zh-CN/model_zoo.md [hunk 2]
```diff
@@ -2,156 +2,144 @@
...
+  * `whole` 模式 （全图模式）:测试的配置文件字段 `test_cfg` 会是 `dict(mode='whole')`. 在这个模式下，全图会被直接输入到网络中。
```
Comment (Junjun2016) [inline docs_zh-CN/model_zoo.md:14]: 全图会被直接输入到网络中 --> 全图会被直接输入到网络中进行推理

File: docs_zh-CN/model_zoo.md [hunk 3]
```diff
@@ -2,156 +2,144 @@
...
-    In this mode, multiple patches will be cropped from input image, passed into network individually.
-    The crop size and stride between patches are specified by `crop_size` and `stride`.
-    The overlapping area will be merged by average
-
-  * `whole` mode: The `test_cfg` will be like `dict(mode='whole')`.
-
-    In this mode, the whole imaged will be passed into network directly.
-
-    By default, we use `slide` inference for 769x769 trained model, `whole` inference for the rest.
-* For input size of 8x+1 (e.g. 769), `align_corner=True` is adopted as a traditional practice.
```
Comment (Junjun2016) [inline docs_zh-CN/model_zoo.md:25]: The translation of this paragraph seems to have been missed.

**Comment by hellock** [comment]
Task linked: [CU-9hv8gg 中文文档 Model Zoo](https://app.clickup.com/t/9hv8gg)

**Review by MengzhangLI — COMMENTED** [review]
> Line 163 "Note: The output stride of DeepLabV3+ is 8."

Does this line need to be translated?

**Pull request merged**